e2e_test/debug-fee-currency/lib.sh

```diff
@@ -13,16 +13,16 @@ set -xeo pipefail
 # returns:
 # 	deployed fee-currency address
 function deploy_fee_currency() {
-	(
-		local fee_currency=$(
+	(set -e;
+		local fee_currency=$(set -e;
```

Comment:
We have no common bash style, but my though is we should have -e set everywhere, so this is unnecessary noise. But since we don't have much bash library code it's fine either way.

Reply:
This was a mistake on my part, I think they can be removed.